protocols/dcutr/CHANGELOG.md

```diff
@@ -2,8 +2,15 @@
 
 - Raise MSRV to 1.65.
   See [PR 3715].
+ - Remove deprecated items. See [PR 3700].
+   * `libp2p_dcutr::inbound::UpgradeError::InvalidAddrs`
+   * `libp2p_dcutr::outbound::UpgradeError::InvalidAddrs`
+   * `libp2p_dcutr::InboundUpgradeError`
+   * `libp2p_dcutr::OutboundUpgradeError`
+   * `libp2p_dcutr::behaviour::*`
```

Comment:
```suggestion
```

I am tempted to exclude this, just to keep the changelog tidy. If users disallow warnings, they can just read over this and otherwise, follow the link to learn more.